### PR TITLE
[Med Prices NG] Scrape data on brand names

### PR DESCRIPTION
**Pull request by RyanSept**
#### What does this PR do?
Creates scraper that collects medicine brand names and stores them on an AWS S3 bucket.
#### Description of Task to be completed?
Create and set up brand name scraper on morph.io
#### How should this be manually tested?
Go to https://morph.io/RyanSept/healthtools_ng

### CODE REVIEW

File: healthtools/config.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import os
+
+AWS = {
+    "aws_access_key_id": os.getenv("MORPH_AWS_ACCESS_KEY_ID"),
+    "aws_secret_access_key": os.getenv("MORPH_AWS_SECRET_KEY"),
+    "region_name": os.getenv("MORPH_AWS_REGION"),
```
Comment (DavidLemayian) [inline healthtools/config.py]: Let's just default this to `eu-west-1`. We don't intend to use any other region in the near future.

File: healthtools/scrapers/brand_names.py [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+from datetime import datetime
+from healthtools.config import AWS
+import requests
+import boto3
+import re
+import json
+import hashlib
+
+
+class BrandNamesScraper(object):
```
Comment (DavidLemayian) [inline healthtools/scrapers/brand_names.py]: Let's change this to `MedBrandNamesScraper` to be clear that it's for Medicines.

**Review by DavidLemayian — CHANGES_REQUESTED** [review]
This looks good! Well done @RyanSept. Just two slight tweaks. :)

**Comment by RyanSept** [comment]
Changes implemented.

**Review by DavidLemayian — APPROVED** [review]

**Pull request merged**